src/clusterfuzz/_internal/metrics/monitoring_metrics.py

```diff
@@ -32,6 +32,19 @@
         monitor.BooleanField('bad_build'),
     ])
 
+JOB_BUILD_AGE = monitor.CumulativeDistributionMetric(
+    'task/fuzz/job/build_age',
```

Comment:
`task/fuzz/job/build_age` makes it sound like this metric is only collected for `fuzz_task`